linalg/petscinternals.hpp

```diff
@@ -18,21 +18,21 @@
 // Error handling
 // Prints PETSc's stacktrace and then calls MFEM_ABORT
 // We cannot use PETSc's CHKERRQ since it returns a PetscErrorCode
-#define PCHKERRQ(obj,err) do {                                                   \
-     if ((err))                                                                  \
-     {                                                                           \
-        PetscError(PetscObjectComm((PetscObject)(obj)),__LINE__,_MFEM_FUNC_NAME, \
-                   __FILE__,(err),PETSC_ERROR_REPEAT,NULL);                      \
-        MFEM_ABORT("Error in PETSc. See stacktrace above.");                     \
-     }                                                                           \
+#define PCHKERRQ(obj,err) do {                                                         \
+     if ((err))                                                                        \
+     {                                                                                 \
+        (void)PetscError(PetscObjectComm((PetscObject)(obj)),__LINE__,_MFEM_FUNC_NAME, \
+                   __FILE__,(err),PETSC_ERROR_REPEAT,NULL);                            \
```

Comment:
```suggestion
                   __FILE__,(PetscErrorCode)(err),PETSC_ERROR_REPEAT,NULL);                            \
```
Good to add a cast here as-well?